### PR TITLE
Adjusted incorrect URL

### PR DESCRIPTION
**Pull request by justinegeffen**
Updated version number in URLs.

### CODE REVIEW

File: platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 - 
   Environment variable:            '`TOWER_ENABLE_AWS_SSM`'
   Description: > 
-    Enable Seqera configuration value retrieval from [AWS Parameter Store](https://docs.seqera.io/platform/24.1.0/enterprise/configuration/aws_parameter_store). 
+    Enable Seqera configuration value retrieval from [AWS Parameter Store](https://docs.seqera.io/platform/24.1/enterprise/configuration/aws_parameter_store).
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml:15]:
```suggestion
    Enable Seqera configuration value retrieval from [AWS Parameter Store](https://docs.seqera.io/platform/latest/enterprise/configuration/aws_parameter_store). 
```

File: platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml [hunk 2]
```diff
@@ -27,5 +27,5 @@
 - 
   Environment variable:            '`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES`'
   Description: > 
-    Enable the [stepped launch form](https://docs.seqera.io/platform/24.1.0/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace. 
+    Enable the [stepped launch form](https://docs.seqera.io/platform/24.1/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml:30]:
```suggestion
    Enable the [stepped launch form](https://docs.seqera.io/platform/latest/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace. 
```

File: platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml [hunk 3]
```diff
@@ -27,5 +27,5 @@
 - 
   Environment variable:            '`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES`'
   Description: > 
-    Enable the [stepped launch form](https://docs.seqera.io/platform/24.1.0/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace. 
+    Enable the [stepped launch form](https://docs.seqera.io/platform/latest/launch/launchpad#launch-form) in the workspaces specified. Set empty (`TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=`) to enable the new launch form in all workspaces, or provide a comma-separated list of workspace IDs to enable the form per workspace.
```
Comment (christopher-hakkaart) [inline platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml:30]: I find "Set empty" a little confusing. Does this mean to set `TOWER_STEPPED_LAUNCH_FORM_ALLOWED_WORKSPACES=` to empty?

Comment (llewellyn-sl) [reply platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml:30]: Literally an empty value, but must be set to enable in all workspaces. If you just leave the env var out altogether, that's a `null` value so then the launch form is _not_ enabled.

Comment (christopher-hakkaart) [reply platform_versioned_docs/version-24.1/enterprise/configuration/configtables/features_env.yml:30]: Ahh, that makes sense.

**Review by christopher-hakkaart — APPROVED** [review]

**Pull request merged**